### PR TITLE
chore: translations for login.properties

### PR DESCRIPTION
**Pull request by ui-okta**
## Description
Auto copy of latest translations by Okta-smartling script.
### Bacon task details
- Task ID: 0d8789e906b742298a6b2cce9380ea6f 
- Requested by: evelyn.teo@okta.com

## Notification sent to
@okta/nxt @haishengwu-okta @nikhilvenkatraman-okta @aarongranick-okta @mauriciocastillosilva-okta

### CODE REVIEW

File: packages/@okta/i18n/src/properties/login_da.properties [hunk 1]
```diff
@@ -221,7 +221,7 @@ factor.oktaVerifyPush = Okta Verify
 factor.push.description = Brug en pushmeddelelse sendt til den mobile app
 factor.duo = Duo Security
 factor.duo.description = Brug pushmeddelelse, SMS eller taleopkald til godkendelse.
-factor.sms = SMS Authentication
+factor.sms = SMS godkendelse
```
Comment (evelynteo-okta) [inline packages/@okta/i18n/src/properties/login_da.properties:224]: @anipendakur-okta This is the fix for https://oktainc.atlassian.net/browse/OKTA-402542.

File: packages/@okta/i18n/src/properties/login_ru.properties [hunk 2]
```diff
@@ -221,7 +221,7 @@ factor.oktaVerifyPush = Okta Verify
 factor.push.description = \u0418\u0441\u043F\u043E\u043B\u044C\u0437\u0443\u0439\u0442\u0435 push-\u0443\u0432\u0435\u0434\u043E\u043C\u043B\u0435\u043D\u0438\u0435, \u043E\u0442\u043F\u0440\u0430\u0432\u043B\u0435\u043D\u043D\u043E\u0435 \u043D\u0430 \u043C\u043E\u0431\u0438\u043B\u044C\u043D\u043E\u0435 \u043F\u0440\u0438\u043B\u043E\u0436\u0435\u043D\u0438\u0435.
 factor.duo = Duo Security
 factor.duo.description = \u0418\u0441\u043F\u043E\u043B\u044C\u0437\u0443\u0439\u0442\u0435 push-\u0443\u0432\u0435\u0434\u043E\u043C\u043B\u0435\u043D\u0438\u0435, SMS \u0438\u043B\u0438 \u0433\u043E\u043B\u043E\u0441\u043E\u0432\u043E\u0439 \u0437\u0432\u043E\u043D\u043E\u043A \u0434\u043B\u044F \u0430\u0443\u0442\u0435\u043D\u0442\u0438\u0444\u0438\u043A\u0430\u0446\u0438\u0438.
-factor.sms = SMS Authentication
+factor.sms = SMS-\u0430\u0443\u0442\u0435\u043D\u0442\u0438\u0444\u0438\u043A\u0430\u0446\u0438\u044F
```
Comment (evelynteo-okta) [inline packages/@okta/i18n/src/properties/login_ru.properties:224]: @anipendakur-okta This is a related Russian fix for the Danish bug here https://oktainc.atlassian.net/browse/OKTA-402542.

File: packages/@okta/i18n/src/properties/login_cs.properties [hunk 3]
```diff
@@ -59,116 +59,116 @@ errors.E0000047 = Byl prekrocen maxim\u00E1ln\u00ED pocet po\u017Eadavku. Zkuste
 errors.E0000079 = Operace nen\u00ED povolena. Chcete-li pokra\u010Dovat, obnovte str\u00E1nku.
 
 # ERROR MESSAGE TRANSLATIONS
-errors.E0000001 = Platnost Api selhala: {0}
+errors.E0000001 = Ov\u011B\u0159en\u00ED rozhran\u00ED API se nezda\u0159ilo: {0}
```
Comment (evelynteo-okta) [inline packages/@okta/i18n/src/properties/login_cs.properties:62]: FYI. All updates to existing Czech translations is related to quality improvement for legacy translation.

**Review by nikhilvenkatraman-okta — APPROVED** [review]
Base branch should be 5.7 instead of master

**Comment by nikhilvenkatraman-okta** [comment]
I cherry-picked these commits onto a new PR based off the `5.7` branch https://github.com/okta/okta-signin-widget/pull/2054

**Comment by aarongranick-okta** [comment]
merged in https://github.com/okta/okta-signin-widget/pull/2054

**Pull request closed without merging**